Assignment4/Order.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Assignment4
+{
+    public class Order
+    {
+        [Column("orderid")]
+        public int Id { get; set; }
+
+        [Column("orderdate")]
+        public DateTime Date { get; set; }
+
+        [Column("requireddate")]
+        public DateTime Required { get; set; }
+
+        [Column("shippeddate")]
+        public DateTime Shipped { get; set; }
+        public int Freight { get; set; }
+        public OrderDetails OrderDetails { get; set; }
+        public string ShipName { get; set; }
+        public string ShipCity { get; set; }
+    }
```

Comment:
I think we need to talk about this as a team before merging it with master. While this is certainly one way to do mapping, the teacher said its not the **proper** way In other words, it can get messy very fast since we have the database domain creeping in on the c# program domain. We need to practice separation of concern so database stuff on one project, c# code one one project and make c# depend on database project.